src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -10035,8 +10004,6 @@ void InitializeOperatingMode(EnergyPlusData &state,
     Real64 SPTempHi;         // thermostat setpoint high
     Real64 SPTempLo;         // thermostat setpoint low
     int NumTU;               // loop counter, number of TU's in list
-    int TUIndex;             // index to TU
-    int ThisZoneNum;         // index to zone number where TU is located
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10038]:(style),[variableScope],The scope of the variable 'TUIndex' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10039]:(style),[variableScope],The scope of the variable 'ThisZoneNum' can be reduced.